eth-connector/src/connector_impl.rs

```diff
@@ -118,11 +118,6 @@ impl EthConnector {
             self.prover_account,
         );
 
-        // Do not skip bridge call. This is only used for development and diagnostics.
-        let skip_bridge_call = false.try_to_vec().unwrap();
```

Comment:
Any reason to do so? This part of the code with a long story. And it's how currently **work in production**.

Reply:
As I understand, because of this new [struct](https://github.com/aurora-is-near/aurora-eth-connector/pull/53/files#diff-44db426fb03c6e9475319fbddd1a63834f936e7775cce69c061e30bef5095069R57) has been introduced. It's needless now.

Reply:
Ah I see, because **trait** was changed: https://github.com/aurora-is-near/aurora-eth-connector/pull/53/files#diff-c4711c94c15be12315d6ea2a1e50f1ff863aca788dfdadc09d4b08cec59071d1R38